### PR TITLE
DEV-169 Coupon endpoint

### PR DESCRIPTION
**Pull request by patrickvenetz**
This Pull Requests adds everything about the coupon endpoint into the API documentation.

### CODE REVIEW

**Comment by lossefanya** [comment]
:+1:

File: api/coupon.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+---
+
+# Coupon
+
+All API requests to `https://api.parku.ch/v4/coupon` need a __private key__.
+
+## Test a coupon
+
+```sh
+$ curl {{ site.parku.api }}/coupon/Zm9vYmFyMTIz\?action\=test\&amount\=42 \
```
Comment (dabio) [inline api/coupon.md]: You can either escape or use quotes for the url.

**Comment by dabio** [comment]
:+1:

File: api/coupon.md (hunk 1 above)
Comment (patrickvenetz) [inline api/coupon.md]: :heart:

**Pull request merged**